src/ck-core/debug-charm.C

```diff
@@ -67,7 +67,7 @@ void CpdBeforeEp(int ep, void *obj, void *msg) {
     entry.obj = obj;
     if (msg != NULL) {
       entry.msg = msg;
```

Comment:
The body of this if-statement can be simplified to 'entry.msg = CkReferenceMsg(msg);' now

Reply:
Good call, done.